<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollapsibleArea</class>
 <widget class="QWidget" name="CollapsibleArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>39</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="3" alignment="Qt::AlignmentFlag::AlignVCenter">
    <widget class="QPushButton" name="clearButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/circle-xmark.svg</normaloff>:/icons/circle-xmark.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="changedFiltersLabel">
     <property name="styleSheet">
      <string notr="true">color: rgb(110, 29, 114);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea {
	border: none;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="expandButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	icon: url(:/icons/angle-right.svg);
}

QPushButton:checked {
	icon: url(:/icons/angle-down.svg);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="labelPlaceholder">
      <property name="leftMargin">
      <number>0</number>
      </property>
      <property name="topMargin">
      <number>0</number>
      </property>
      <property name="rightMargin">
      <number>0</number>
      </property>
      <property name="bottomMargin">
      <number>0</number>
      </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
